js/Utils.js

```diff
@@ -44,6 +47,18 @@ export default {
     }
     return cssStr;
   },
+  styleToButton(style: Object): string {
+    let cssStr = '';
+    if (style) {
+      if (style['button-width']) {
```

Comment:
I'm not sure it's best to have a direct height and width for the style. Currently style has a size property with 3 options, small, medium, and large. I think that might work better has the structured content schema needs to work for more than just CSS; it has to work for mobile apps, Facebook Messenger, etc. that may or may not have the same concept of CSS pixel height

Reply:
hi @kelvingraddick ,  the current options for size are for text only, the default hardcoded button size (35px) is much bigger than "large". There are no exact requirements for the button size property...

Reply:
See other/new comment.